rest/models/conditions_test.go

```diff
@@ -0,0 +1,36 @@
+package models_test
+
+import (
+	"testing"
+
+	"github.com/polygon-io/client-go/rest/models"
+)
+
+func TestListConditionsParams(t *testing.T) {
+	assetClass := models.AssetClass("Equity")
+	dataType := models.DataType("EndOfDay")
+	id := int64(1)
+	sip := models.SIP("CTA")
```

Comment:
In almost all cases, you can directly use constants from the models package instead of type casting strings, e.g. `sip := models.SIP("CTA")` can be changed to `sip := models.CTA` (see [here](https://github.com/polygon-io/client-go/blob/master/rest/models/types.go) for all the options)